components/gitpod-db/src/typeorm/user-db-impl.ts

```diff
@@ -356,13 +376,26 @@ export class TypeORMUserDBImpl implements UserDB {
         return entry.filter((te) => includeDeleted || !te.deleted);
     }
 
-    protected mapUserToDBUser(user: User): DBUser {
-        const dbUser = user as DBUser;
-        // Here we need to fill the pseudo column 'user' in DBIdentity (see there for details)
-        dbUser.identities.forEach((id) => (id.user = dbUser));
+    private mapUserToDBUser(user: User): DBUser {
```

Comment:
the two methods `mapUserToDBUser` and `mapDBUserToUser` are only used on this layer. A DBUser object should never leave this data access layer.